### DISCUSSION TITLE
What is the "Control Button"?

### DISCUSSION THREAD

**Discussion by madmacks59** [HomeSpan/HomeSpan#204] [Q&A] [answered]
I'm sure it's obvious to everyone but me but what does the following mean?

"To enter Device Configuration Mode, press and hold the Control Button for 3 seconds."

Is that a Reference to the EN or Boot button on the ESP32?

**Comment by HomeSpan** [comment, marked as answer]
https://github.com/HomeSpan/HomeSpan/blob/master/docs/GettingStarted.md

**Comment by madmacks59** [comment]
DERP...RTFM, Sorry...I skimmed that doco and missed it.
